docs/pulseload.md

```diff
@@ -23,41 +22,47 @@ username and password. Remember these as you'll use them in the next step.
 Unfortunately, **Pulse** doesn't support creating queues with a guest account, so
 this step is necessary.
 
-### 2. Environment Variable
-
-If your **Pulse User** was username: `foo` and password: `bar`, your config
-string would be:
+If your **Pulse User** was username: `foo` and password: `bar`, your Pulse URL
+would be:
 
 `amqp://foo:bar@pulse.mozilla.org:5671/?ssl=1`
 
-### 3. Read Pushes
+### 2. Read Pushes
+
+You will need the root URL for the Taskcluster deployment, such as `https://firefox-ci-tc.services.mozilla.com`.
```

Comment:
This line threw me since it *currently* should be "https://taskcluster.net".  I suppose in the future the value you set here is going to be the default, I take it?

Reply:
Haha.  Fair enough, yes.